lib/email_doc/document.rb

```diff
@@ -5,22 +5,22 @@ module EmailDoc
   class Document
     extend Forwardable
 
-    attr_reader :mail, :context
+    attr_reader :mail, :context, :ex
```

Comment:
nits: I don't think that "ex" is good naming.  Because most of programmers understand "ex" is an instance of Exception (not Example).

But it is nits issue, so I will merge this PR now~!